README.md

```
@@ -27,26 +30,30 @@ The API documentation can be found [here](https://mops.one/sha2/docs/lib) on Mop
-* [Dfinity forum](https://forum.dfinity.org/)
+- [OpenChat group](https://oc.app/2zyqk-iqaaa-aaaar-anmra-cai)
+- [Twitter](https://twitter.com/mr_research_ag)
+- [Dfinity forum](https://forum.dfinity.org/)
 
 ## Usage
+
 ### Install with mops
 
 You need `mops` installed. In your project directory run:
+
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)